### DISCUSSION TITLE
Email Sent With Error

### DISCUSSION THREAD

**Discussion by ZeaZuniga** [emailjs-com/help#49] [Q&A]
I got emailjs all plugged into my reactTS project, and got the test and first emails sent successfully. But when I sent subsequent tests, they would not appear for at least 30 minutes, if at all. I have switched the connected email to test and each is resulting in a long delay before receiving the email even though the email-history on the dashboard says it has already been sent. Is this to be expected even though the first email was sent within a few seconds?

Additionally, upon receiving the first test email, I got a follow up email from "Mail Delivery Subsystem saying that the address was not found. The issue is that the address it was looking for switched out my original gmail account for an outlook account. Did I do something wrong? What causes the system to switch the email service I assigned it to?

**Comment by xr0master** [comment]
Hi,

The OK means that the email has been verified and successfully transferred to your service. Your service has accepted the processing request. Unfortunately, we have no control over the last steps.

If you use the Outlook service, then your Outlook address will be used to send emails, which you connected to the service.

**Reply by ZeaZuniga** [reply]
I ended up getting all the test emails I had sent, but after several hours. I understand that beyond the verification and processing it is beyond your control, but is that to be expected? 

That's the weird part. I signed up with an outlook email, but the service I'm using for all the tests is through google.

**Reply by xr0master** [reply]
No, emails usually arrive within a short time. However, large companies also have issues, although they fix it quickly enough. Does this problem still exist?

> That's the weird part. I signed up with an outlook email, but the service I'm using for all the tests is through google.

I see, you can change the recipient address in the template.

**Reply by ZeaZuniga** [reply]
That's what it was!! Thanks for letting me know to change the sender/recipient address. Once I changed that, everything worked smoothly and I got the emails within seconds. I'm guessing that with the incorrect sender email, it kept trying to send and then went through each email service. Sorry for the hassle, but thank you for your help!
